setup.py

```diff
@@ -16,17 +16,13 @@
     "pytest-django==4.5.0",
     "pytest-cov==3.0.0",
     "pytest-pythonpath==0.7.3",
```

Comment:
this may be for another PR, but pytest and the pytest plugins all have more recent releases which would be nice to use.

also pytest-pythonpath is obsolete as of pytest 7.0

Reply:
Thanks, opened https://github.com/wagtail/wagtail-factories/issues/86